framework/resources/Datasources/json_storage/MD_4707/MD_4707_import_tables.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+source conf/drillTestConfig.properties
+
+hadoop_folder=${DRILL_TESTDATA}/MD_4707
+test_data=MD_4707_test_data.json
+maprdb_table=MD_4707_json_MapR_DB_table
```

Comment:
Instead of the /MD_4707/MD_4707_json_MapR_DB_table dir, can we use something else more clear? Say hive/maprdb/json/<table_name> directory?

Also i would suggest you use tpch or tpcds dataset, that way we do not introduce another new dataset into the framework. All we need is to create a Hive external table for an existing tpch/tpcds maprdb table. I think I already have few in.

Reply:
Usually I prefer to use my own tables if they don't take a lot of space. And re-using of data may result in unexpected failures, there were such incidents.

Reply:
External tables should be safe. I want to eventually get rid of these small random datasets and work on standardizing on data sets and tests. So please update the tests to use the tpch/tpcds maprdb tables which are created in a different suite.

Reply:
Renamed the folder to hive/maprdb/json/. 
As for the tpch/tpcds maprdb tables, they all have column names in upper case and cannot be used with Hive due to the bug: https://maprdrill.atlassian.net/browse/HIVE-298. @Agirish  Could you suggest any workaround?